tests/integration/test_backup_restore_s3/test_throttling.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+from minio.deleteobjects import DeleteObject
+
+from helpers.cluster import ClickHouseCluster
+from helpers.s3_tools import list_s3_objects
+from helpers.utility import random_string
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance(
+    "node",
+    main_configs=[
+        "configs/throttling_disks.xml",
+    ],
+    stay_alive=True,
+    with_minio=True,
+)
+
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    try:
+        cluster.start()
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+@pytest.mark.parametrize(
+    "storage_policy", ["default", "s3_native_copy", "s3_no_native_copy"]
+)
+@pytest.mark.parametrize("allow_s3_native_copy", [0, 1])
+def test_backup_scheduler_settings(
+    started_cluster, storage_policy, allow_s3_native_copy
+):
+
+    minio = cluster.minio_client
+    s3_objects = list_s3_objects(minio, cluster.minio_bucket, prefix="")
+    assert (
+        len(
+            list(
+                minio.remove_objects(
+                    cluster.minio_bucket,
+                    [DeleteObject(obj) for obj in s3_objects],
+                )
+            )
+        )
+        == 0
+    )
+
+    table_name = f"test_s3_storage_class_{storage_policy}_{allow_s3_native_copy}"
+
+    node.query(f"CREATE OR REPLACE RESOURCE network_read (READ DISK {storage_policy})")
+    node.query(f"CREATE OR REPLACE WORKLOAD backup SETTINGS max_requests = 10")
```

Comment:
A bit more realistic workload hierarchy would be:
```
CREATE OR REPLACE WORKLOAD all SETTINGS max_requests = 100
CREATE OR REPLACE WORKLOAD backup PARENT all SETTINGS max_requests = 10
CREATE OR REPLACE WORKLOAD default PARENT all
```
This would limit backup to 10 requests while all activities (including backup) will be limited to 100 requests.